internal/cmd/diag.go

```diff
@@ -39,7 +38,7 @@ func ExecInUvm(ctx context.Context, vm *uvm.UtilityVM, req *shimdiag.ExecProcess
 
 // ExecInShimHost is a helper function used to execute commands specified in `req` in the shim's
 // hosting system.
-func ExecInShimHost(ctx context.Context, req *shimdiag.ExecProcessRequest) (int, error) {
+func ExecInShimHost(ctx context.Context, req *CmdProcessRequest) (int, error) {
```

Comment:
I feel like I'm getting deja-vu on another discussion on this naming, but we should probably remove the Shim from this name, can be another PR. Don't think this package should have any mention of it.